language/go/embed.go

```diff
@@ -30,18 +30,26 @@ import (
 // embedResolver maps go:embed patterns in source files to lists of files that
 // should appear in embedsrcs attributes.
 type embedResolver struct {
-	// files is a list of embeddable files and directories, stored in
-	// depth-first pre-order (nearly the same order as filepath.Walk, except
-	// files in the top-level directory appear before subdirectories.
-	files []embeddableFile
+	// files is a list of embeddable files and directory trees, rooted in the
+	// package directory.
+	files []*embeddableFile
 }
 
 type embeddableFile struct {
```

Comment:
Is this still best thought of as an `embeddableFile` or is it now an `embeddableNode` since it's a tree?

Reply:
Or maybe `embeddableFileNode`